### PR TITLE
Phone bottom bar and some styling

### PR DESCRIPTION
**Pull request by chuyiting**
The bottom bar does not work with the ace editor. It should disappear when you scroll down, anyone can help solve this question??

### CODE REVIEW

**Comment by benclmnt** [comment]
hi, i don't think it is necessary to change global state just to incorporate styling
We can work out some kind of carousel if you want, that way is nicer than changing routes imo.

see #16 .

wdyt?

File: public/index.html [hunk 1]
```diff
@@ -24,6 +24,16 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
+    <script src="https://code.jquery.com/jquery-3.4.1.js"></script>
+    <script type="text/javascript">
+      var zero = 0;
+      $(document).ready(function() {
```
Comment (benclmnt) [inline public/index.html]: what is this for?
why add jquery? you can just put your javascript code in some hooks right?

**Review by benclmnt — DISMISSED** [review]
May i know why you literally copy Workspace to WorkspaceCom?

I am suggesting of using a carousel that would not affect how state is managed right now in the computer view, so that we dont need to handle states based on device view port.

Other than that, there is just some minor conflicts that needs to be resolved when merging.

wdyt?

File: tsconfig.json [hunk 2]
```diff
@@ -1,11 +1,7 @@
 {
   "compilerOptions": {
-    "target": "es5",
-    "lib": [
-      "dom",
-      "dom.iterable",
-      "esnext"
-    ],
+    "target": "es2016",
```
Comment (benclmnt) [inline tsconfig.json]: any reason as to change this to es2016?

**Comment by benclmnt** [comment]
pls resolve the conflicts so that i can merge it with the develop branch. thx

**Comment by chuyiting** [comment]
> May i know why you literally copy Workspace to WorkspaceCom?
> 
> I am suggesting of using a carousel that would not affect how state is managed right now in the computer view, so that we dont need to handle states based on device view port.
> 
> Other than that, there is just some minor conflicts that needs to be resolved when merging.
> 
> wdyt?

**Pull request closed without merging**